pyrdp/player/LiveEventHandler.py

```diff
@@ -139,7 +217,26 @@ def onFileDownloadRequested(self, file: File, targetPath: str, dialog: FileDownl
         self.downloadFiles[remotePath] = targetFile
         self.downloadDialogs[remotePath] = dialog
 
-        pdu = PlayerFileDownloadRequestPDU(self.layer.getCurrentTimeStamp(), parent.deviceID, file.getFullPath())
+        pdu = PlayerFileDownloadRequestPDU(self.layer.getCurrentTimeStamp(), parent.deviceID, remotePath)
+        self.layer.sendPDU(pdu)
+
+    def onDirectoryDownloadRequested(self, directory: Directory, targetPath: str, dialog: FileDownloadDialog):
```

Comment:
since this method got way bigger docstring would help

Reply:
Will do! I'll add more documentation to the flow of the module in general.